wordpress-mobile/WordPress-iOS
# Clean up test extensions for UI tests

This removes unused test extensions from a past implementation of our UI tests. These are now covered in the page objects under `/Screens`, aside from the shared `isIpad` function that can now be called from within the page objects.

To test:

* Start the simulator.
* Confirm all of the tests in WordPressUITests still pass.
